src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -99,11 +99,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         /** @var KernelInterface $kernel */
         $kernel = $this->getApplication()->getKernel();
-        $targetArg = rtrim($input->getArgument('target'), '/');
-
-        if (!$targetArg) {
-            $targetArg = $this->getPublicDirectory($kernel->getContainer());
-        }
+        $targetArg = rtrim($input->getArgument('target') ?? $this->getPublicDirectory($kernel->getContainer()), '/');
```

Comment:
We try to be very conservative on maintenance branches. This change is not doing the same thing than before.

Something like this should be enough:
```suggestion
        $targetArg = rtrim($input->getArgument('target') ?? '', '/');
        
        if (!$targetArg) {
            $targetArg = $this->getPublicDirectory($kernel->getContainer());
        }
```